package.json

```diff
@@ -30,11 +30,11 @@
   "dependencies": {
     "debounce-on": "^0.0.0",
     "eventemitter2": "^4.0.0",
-    "suggest-it": "^0.0.1"
+    "suggest-it": "^0.0.2"
   },
   "devDependencies": {
-    "angular": "^1.6.3",
-    "angular-mocks": "^1.6.3",
+    "angular": "^1.6.6",
+    "angular-mocks": "^1.6.6",
```

Comment:
These are `devDependencies`. Is Firefox really blocking extensions for `devDependencies`?